aait/backend/group-2A/Persistence/Repositories/LikeRepository.cs

```diff
@@ -0,0 +1,42 @@
+using Application.Contracts.Persistance;
+using Application.DTO.UserDTO;
+using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace Infrastructure.Persistance.Repository;
+
+public class LikeRepository : ILikeRepository{
+    private readonly SocialSyncDbContext _dbContext;
+    private readonly DbSet<Like> _dbLikes;
+
+    public LikeRepository(SocialSyncDbContext context){
+        _dbContext = context;
+        _dbLikes = context.Set<Like>();
+    }
+    public async Task<List<UserDto>> Likers(int id){
```

Comment:
In C#, it's common to use PascalCase for method names. Consider renaming the method Likers to GetLikers to follow standard naming conventions.